File: src/main/java/codereview/simpleorder/repository/command/OrderRepository.java
package codereview.simpleorder.repository.command;

import codereview.simpleorder.domain.item.Clothes;
import codereview.simpleorder.domain.order.Order;
import org.springframework.data.jpa.repository.JpaRepository;

public interface OrderRepository extends JpaRepository<Order, Long> {
}

File: micro-services/order/src/main/kotlin/io/philo/shop/repository/OrderRepository.kt
package io.philo.shop.repository

import io.philo.shop.domain.core.OrderEntity
import org.springframework.data.jpa.repository.JpaRepository

interface OrderRepository : JpaRepository<OrderEntity, Long> {

    fun findAllByIdIn(orderIds: List<Long>): List<OrderEntity>
}